src/navigation/ScrollPagerWrapper.js

```diff
@@ -4,19 +4,22 @@ import ScrollPager from '../helpers/ScrollPager';
 import { ScrollPositionContext } from './ScrollPositionContext';
 import ViewPagerAdapter from './ViewPagerAdapter';
 
-export default forwardRef(function ScrollPagerWrapper(props, ref) {
-  const position = useSharedValue(props.initialScrollPosition ?? 0);
+export default forwardRef(function ScrollPagerWrapper(
+  { initialScrollPosition, useViewPagerAdaptor = android, ...rest },
+  ref
+) {
+  const position = useSharedValue(initialScrollPosition ?? 0);
```

Comment:
adding this as a note to do some QA in the swaps view, the component is working fine from what I can see in the app